### PR TITLE
Added to the readme file

### PR DESCRIPTION
**Pull request by liamwarfield**
Added fixes for some common problems that were encountered in the first ACM meeting this semester

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,11 +30,14 @@ Setup the application:
 
     $ cp development.ini.sample development.ini
     $ gearbox setup-app
+If `gearbox setup-app` fails try specifying `pip3` in the previos two steps
```
Comment (sumnerevans) [inline README.md]: 1. s/previos/previous
    2. "... fails, try ..."
                 ^
    3. "... steps."
                 ^

File: README.md [hunk 2]
```diff
@@ -28,11 +30,14 @@ Setup the application:
...
 
-Finally, serve the application:
+Serve the application:
 
     $ gearbox serve --reload --debug
 
+Finaly, go into your web browser and go to localhost:8080 to view the website.
```
Comment (sumnerevans) [inline README.md]: 1. "... go ***to*** your web browser and **navigate** to ..."
2. Wrap "localhost:8080" in backticks to make it render as monospace:

        ... to `localhost:8080` to ...

File: README.md [hunk 3]
```diff
@@ -70,10 +79,12 @@ Setup the application:
     $ cp development.ini.sample development.ini
     $ gearbox setup-app
 
-Finally, serve the application:
+Serve the application:
 
     $ gearbox serve --reload --debug
 
+Finaly, go into your web browser and go to localhost:8080 to view the website.
```
Comment (sumnerevans) [inline README.md]: Same as above.

Comment (liamwarfield) [reply README.md]: Fixxed it

File: README.md (hunk 1 above)
Comment (sumnerevans) [reply README.md]: Please see my first note 1. There's a typo.

**Comment by liamwarfield** [comment]
I added the suggestions

File: README.md [hunk 4]
```diff
@@ -28,11 +30,14 @@ Setup the application:
 
     $ cp development.ini.sample development.ini
     $ gearbox setup-app
+If `gearbox setup-app` fails, try specifying `pip3` in the previos two steps.
```
Comment (sumnerevans) [inline README.md]: previos -> previous

**Pull request merged**